Formula/aerc.rb

```diff
@@ -18,7 +18,7 @@ class Aerc < Formula
   depends_on "scdoc" => :build
 
   def install
-    system "make", "PREFIX=#{prefix}"
+    system "make", "SHAREDIR=/usr/local/share/aerc", "PREFIX=#{prefix}"
```

Comment:
```suggestion
    system "make", "SHAREDIR=#{opt_pkgshare}", "PREFIX=#{prefix}"
```
We don't want to hardcore `/usr/local` here -- that's not always where Homebrew is installed.

Reply:
Thanks, I've applied these changes